invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -849,32 +851,48 @@
           "items": {"$ref": "#/definitions/agent"}
         },
 
-        "embargo_date": {
-          "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
-          "type": "string",
-          "format": "date-time"
-        },
-
-        "access_condition": {
-          "description": "Conditions under which access to files are granted",
+        "embargo": {
+          "description": "Description of the embargo on the record.",
           "type": "object",
+          "required": ["active", "until", "reason"],
```

Comment:
Minor: Do we need them as required? It often causes problems with partial data (e.g. a user saving an incomplete draft). Perhaps it's fine if the system field always guarantees them to be there independently of what the user submitted.